kafka-rest/src/main/java/io/confluent/kafkarest/ratelimit/RateLimitExceededException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.ratelimit;
+
+import io.confluent.kafkarest.exceptions.StatusCodeException;
+import javax.ws.rs.core.Response;
+
+/**
+ * An exception thrown when the request rate-limit has been exceeded.
+ *
+ * @see RequestRateLimiter#rateLimit(int)
+ */
+public final class RateLimitExceededException extends StatusCodeException {
```

Comment:
Consider merging with the RateLimitGracePeriodExceededException - they are doing the same thing.  (Maybe the Grace period exception needs a rename)

Reply:
I explicitly didn't reuse anything from this rate-limiter in the produce-rate-limiter. I think there's much to be reused, but I didn't want more time (or rather didn't have more time to spend).